NOTICE.txt

```diff
@@ -0,0 +1,8 @@
+Apache MINA SSHD
```

Comment:
IANAL, but my understanding is that this is required since I have copied most of BouncyCastleNonBlockingRandom.java from Apache MINA.

Reply:
This looks correct, though you'll need to actually add the Apache license file here, too. Your naming scheme is below, so you'd have something like `LICENSE.mina.txt`.